cmd/cosign/cli/sign/sign.go

```diff
@@ -107,6 +108,19 @@ func UploadToTlog(ctx context.Context, sv *CertSignVerifier, rekorURL string, up
 		return nil, err
 	}
 	fmt.Fprintln(os.Stderr, "tlog entry created with index:", *entry.LogIndex)
+
+	params := entries.NewGetLogEntryByIndexParams()
+	params.LogIndex = *entry.LogIndex
+
+	resp, err := rekorClient.Entries.GetLogEntryByIndex(params)
```

Comment:
Actually, do we need to make this call? Can we use *entry.LogIndex from line 110?

Reply:
that is the log index and we cannot use that to build the URL to open, or maybe I'm missing something

in the line 110 output: `734594` for example

that we can use to open the record?

Reply:
Ah, you can do: https://rekor.sigstore.dev/api/v1/log/entries?logIndex=734594

Reply:
ahhh so dummy from my side :(

will close this PR, we don't need then :)